addons/sale/models/account_invoice.py

```diff
@@ -51,10 +51,6 @@ def _onchange_partner_id(self):
 
         res = super(AccountMove, self)._onchange_partner_id()
```

Comment:
you can return directly `return super()._onchange_partner_id()`
Note: `partner_shipping_id` has been moved to account recently so you could probably remove this override completely during the forwart-port

Reply:
Just updated it and I'll pay attention during forward-port, thanks for the advice!